app/src/main/res/raw/menu.json

```diff
@@ -628,6 +628,40 @@
 					}
 				]
 			}
+		},
+		{
+			"version": 2018050301,
+			"url": "https://nluug.nl/events/vj18/schedule.xml",
+			"title": "NLUUG Spring 2018",
+			"start": "2018-05-15",
+			"end": "2018-05-15",
+			"metadata": {
+				"icon": "https://nluug.nl/events/vj18/nluug-icon-300.png",
+				"links": [
+					{
+						"url": "https://www.nluug.nl/index.html",
+						"title": "NLUUG Website"
+					},
+					{
+						"url": "https://www.nluug.nl/events/vj18/programma.html",
+						"title": "Conference Program (website)"
+					}
+				],
+				"rooms": [
+					{
+						"name": "Room 1",
+						"latlon": [52.060788, 5.108085]
+					},
+					{
+						"name": "Room 2",
+						"latlon": [52.060788, 5.108085]
+					},
+					{
+						"name": "Room 3",
+						"latlon": [52.060788, 5.108085]
```

Comment:
That's fine for me.  I just added the 'latlon'' keys because the schema required so and only afterwards I saw the rooms were left out previous year. I assume the schedule per room stays available (as they are listed in the XML anyway)?

Reply:
latlon is currently required because it's the purpose of the otherwise optional room section.

So it's better to just omit the room section entirely. I'm pretty sure the schema should allow that, if not that's a bug.

Reply:
Do you expect me to do something now or can you add the entry without the rooms now?

Reply:
The conference is next Tuesday.  Will the entry be included in time?

Reply:
Yeah I was waiting for you to remove the room section.... For example https://github.com/Wilm0r/giggity/pull/58/files doesn't have it and is passing the presubmit/CI just fine.

Or do you want to include them as a general venue location pointer? If so, you can just include one room with regex ".*". But I'm not a fan of that idea either..